lib/src/configuration/file.dart

```diff
@@ -25,8 +26,10 @@ Future<Map> readConfiguration(String path) async {
 /// The default configuration will be returned by the function.
 Future<Map> writeDefaultConfiguration(String path) async {
   // Get the contents of the resource
-  var resource = new Resource('package:speakeasy/src/configuration/default.yml');
-  var contents = await resource.readAsString();
+  var packageUri = Uri.parse('package:speakeasy/src/configuration/default.yml');
```

Comment:
Why not use https://pub.dartlang.org/packages/resource ?

Reply:
I was under (probably incorrect) impression, that Resource is somehow deprecated, so when I saw that it did not work in old version of speakeasy I went to replace it. Thinking about it, Resource does not work in browser (see https://github.com/dart-lang/resource/issues/16 ) . In server-side environment it should be OK. Anyway, I'll try you current version on https://github.com/speakeasy-pub